manifests/tasks_service.pp

```diff
@@ -0,0 +1,28 @@
+# = foreman-tasks service
+#
+# Service for the foreman-tasks service, which starts a Dynflow executor
+# for background job processing in Foreman. Not to be confused with the
+# foreman-tasks plugin.
+#
+# === Parameters:
+#
+# $service:: Service name
+#            type:String
+#
+# $ensure:: State the service should be in
+#           type:Enum['absent', 'present']
+#
+# $enable:: Whether to enable the service or not
+#           type:String
+
+class foreman::tasks_service(
+  $service = $::foreman::params::tasks_service,
```

Comment:
I got kind of stuck here - if I remove the params and use `$::foreman::tasks_service`, the spec for `foreman_service_spec` fails - because that spec tests when `$::foreman`  is not set - and this manifest needs it